lapis2/src/test/kotlin/org/genspectrum/lapis/auth/ProtectedDataAuthorizationTest.kt

```diff
@@ -53,13 +56,15 @@ class ProtectedDataAuthorizationTest(@Autowired val mockMvc: MockMvc) {
     fun `given no access key in POST request to protected instance, then access is denied`() {
         mockMvc.perform(postRequestWithBody(""))
             .andExpect(MockMvcResultMatchers.status().isForbidden)
```

Comment:
Just an idea: I like the import of the content() matcher. Can we do the same for status()?